lib/memjs/utils.js

```diff
@@ -5,13 +5,20 @@ var header = require('./header');
 var bufferify = function(val) {
   return Buffer.isBuffer(val) ? val : Buffer.from(val);
 };
+exports.bufferify = bufferify;
 
-exports.makeRequestBuffer = function(opcode, key, extras, value, opaque) {
+exports.addToRequestBuffer = function(opcode, key, extras, value, opaque, buf, _bufTargetWriteOffset) {
```

Comment:
nit: would name this `copyIntoRequestBuffer`, which makes it clear addToRequestBuffer doesn't attempt to resize `buf`

Reply:
done